<launch>
    <!-- <node pkg="map_server" type="map_server" name="map_server_node" args="$(find static_map)/map_file/map.yaml"/>
    <node pkg="rviz" type="rviz" name="Rviz_for_extract_path" args="-d $(find global_planning)/rviz/global_plan_lanecross.rviz" /> -->
    <node pkg="visualization_smart" type="visualization_node" name="visualization_node" output="screen">
        <param name="vis_current_pose" value="true" />

        <param name="topic_current_pose" value="/ndt/current_pose" />
    </node>
</launch>
